src/templates/pages/journal.html

```diff
@@ -11,7 +11,7 @@ <h2 class="journal-title">{{ journal.title }}</h2>
         </div>
         <div class="metadata-item">
             <span class="metadata-key">Homepage:</span>
-            <span class="metadata-value">TODO add homepage to journal model</span>
+            <span class="metadata-value">{{ journal.homepage_url }}</span>
         </div>
```

Comment:
(this should probably be an `<a>` tag and wrapped in `if` if we don't have it sometimes. i can never remember the jinja strip whitespace params so something like this, but not important to get it exactly right this PR since we will have to do style changes later anyway)

```suggestion
        {% if journal.homepage_url %}
        <div class="metadata-item">
            <span class="metadata-key">Homepage:</span>
            <a href="{{ journal.homepage_url }}">
                <span class="metadata-value">{{ journal.homepage_url }}</span>
            </a>
        </div>
        {% endif %}
```

Reply:
you're right, this way it would be cleaner

Reply:
fixed it